src/cljs/main/broadfcui/page/external_importer.cljs

```diff
@@ -48,6 +56,96 @@
          you have signed in and completed the user profile registration step you can start using FireCloud."]]]]))
 
 
+(react/defc DockstoreImporter
+  {:render
+   (fn [{:keys [state this]}]
+     (let [{:keys [wdl load-error]} @state]
+       (cond load-error (this :-render-error)
+             wdl (this :-render-export)
+             :else (spinner "Loading WDL..."))))
+   :component-did-mount
+   (fn [{:keys [props state]}]
+     (let [{:keys [id version]} props]
+       (ajax/call {:url (str "https://staging.dockstore.org:8443/api/ga4gh/v1/tools/"
+                             (js/encodeURIComponent (str "#workflow/" id))
+                             "/versions/"
+                             (js/encodeURIComponent version)
+                             "/WDL/descriptor")
+                   :on-done (fn [{:keys [success? get-parsed-response]}]
+                              (if success?
+                                (swap! state assoc :wdl (:descriptor (get-parsed-response)))
+                                (swap! state assoc :load-error (get-parsed-response false))))})))
+   :-render-error
+   (fn [{:keys [state]}]
+     [:div {}
+      [:div {:style {:margin "0.5rem 0"}} "Error loading WDL:"]
```

Comment:
Since Dockstore's error messages are unhelpful, how about:

"Error loading WDL. Please verify the workflow path and version and ensure this workflow supports WDL."

(there are more error conditions than that, but I think that covers the majority)